mnist/dockerfile/ksonnet-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#FIXME this is set to tf-user by default in order to work around lack of Serviceaccount setting in argo.
+SERVICE_ACCOUNT=${SERVICE_ACCOUNT:-tf-user}
```

Comment:
Do we still need this file it looks like you got rid of the container?

Reply:
actually we don't need this. the env var is set in pod spec.

and this line does not use `export`, the actually command will not recognize this. I leave this unchanged in this PR.